### PR TITLE
build: Update at_auth version in pubspec.yaml

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
- Update at_auth version in pubspec.yaml
- Update the contents in CHANGELOG.md
- Fix dart analyzer issues.

### CODE REVIEW

File: packages/at_auth/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 1.0.6
+
+- build[deps]: Upgraded the following packages:
+  - at_commons to 4.0.5
+  - at_lookup to 3.0.46
+- Introduce new methods for enrollment operations in AtEnrollmentImpl and deprecate the older methods
+- Rename the existing classes related to EnrollmentRequest to increase readability
```
Comment (murali-shris) [inline packages/at_auth/CHANGELOG.md]: improve readability

File: packages/at_auth/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 1.0.6
+
```
Comment (murali-shris) [inline packages/at_auth/CHANGELOG.md]: remove this empty line

**Review by murali-shris — CHANGES_REQUESTED** [review]

File: packages/at_auth/CHANGELOG.md (hunk 2 above)
Comment (sitaram-kalluri) [reply packages/at_auth/CHANGELOG.md]: The empty line is removed

File: packages/at_auth/CHANGELOG.md (hunk 1 above)
Comment (sitaram-kalluri) [reply packages/at_auth/CHANGELOG.md]: Rephrased the lines

**Review by murali-shris — APPROVED** [review]

**Pull request merged**